assets/airports/ksea.json

```diff
@@ -61,151 +69,221 @@
-                "16R": [["NICHY", "A30+"], "ELMAA"]
+                "16L": [],
+                "16C": [],
+                "16R": []
             },
+            "body": [["NICHY", "A30+"], "_NICHY250SEA230", "ELMAA"],
             "exitPoints": {
-                "HQM": ["HQM"],
-                "CVO": ["CVO"]
+                "CVO": ["CVO"],
+                "ELMAA": [],
+                "HQM": ["HQM"]
             },
-            "draw": [["NICHY", "ELMAA"], ["ELMAA", "HQM*"],["ELMAA", "CVO*"]]
+            "draw": [
+                ["NICHY", "_NICHY250SEA230", "ELMAA*"],
+                ["ELMAA", "HQM*"],
+                ["ELMAA", "CVO*"]
+            ]
         },
         "HAROB5": {
             "icao": "HAROB5",
             "name": "Harob Five",
             "rwy": {
-                "16L": ["CAVOB", ["RENBE", "A30+"], "RICHR", "EMRLD", ["HAROB", "A100+"]],
-                "16C": ["OTLIE", ["RENBE", "A30+"], "RICHR", "EMRLD", ["HAROB", "A100+"]],
-                "16R": ["WESET", ["RENBE", "A30+"], "RICHR", "EMRLD", ["HAROB", "A100+"]],
-                "34L": ["CUSBU", ["WUREL", "A31+"], "ATOME", "BREMM", ["HAROB", "A100+"]],
-                "34C": ["NESOE", ["WUREL", "A31+"], "ATOME", "BREMM", ["HAROB", "A100+"]],
-                "34R": ["DODVE", ["WUREL", "A31+"], "ATOME", "BREMM", ["HAROB", "A100+"]]
+                "16L": ["CAVOB", ["RENBE", "A30+"], "RICHR", "EMRLD"],
+                "16C": ["OTLIE", ["RENBE", "A30+"], "RICHR", "EMRLD"],
+                "16R": ["WESET", ["RENBE", "A30+"], "RICHR", "EMRLD"],
+                "34L": ["CUSBU", ["WUREL", "A31+"], "ATOME", "BREMM"],
+                "34C": ["NESOE", ["WUREL", "A31+"], "ATOME", "BREMM"],
+                "34R": ["DODVE", ["WUREL", "A31+"], "ATOME", "BREMM"]
             },
+            "body": [["HAROB", "A100+"]],
             "exitPoints": {
-                "HQM": ["HQM"],
                 "ERAVE": ["ERAVE"],
-                "HISKU": ["FEPOT", "HISKU"]
+                "FEPOT": ["FEPOT"],
+                "HAROB": [],
+                "HISKU": ["FEPOT", "HISKU"],
+                "HQM": ["HQM"]
             },
-            "draw": [["CAVOB", "RENBE"], ["OTLIE", "RENBE"], ["WESET", "RENBE"], ["RENBE", "RICHR", "EMRLD", "HAROB"], ["CUSBU", "WUREL"], ["NESOE", "WUREL"], ["DODVE", "WUREL"], ["WUREL", "ATOME", "BREMM", "HAROB"], ["HAROB", "HQM*"], ["HAROB", "ERAVE*"], ["HAROB", "FEPOT", "HISKU*"]]
+            "draw": [
+                ["CAVOB", "RENBE"],
+                ["OTLIE", "RENBE"],
+                ["WESET", "RENBE"],
+                ["RENBE", "RICHR", "EMRLD", "HAROB"],
+                ["CUSBU", "WUREL"],
+                ["NESOE", "WUREL"],
+                ["DODVE", "WUREL"],
+                ["WUREL", "ATOME", "BREMM", "HAROB"],
+                ["HAROB", "HQM*"],
+                ["HAROB", "ERAVE*"],
+                ["HAROB*", "FEPOT*", "HISKU*"]
+            ]
         },
         "KMORE4": {
```

Comment:
Must have been removed the other day when they added the ATOME1. Cool!

Removed.